docs/device/install.md

```diff
@@ -26,11 +35,20 @@
     Using Gnome DE on latest Ubuntu LTS - only supported variant atm
 
 1. [Install Kolide agent](install.md#install-kolide-agent).
-2. Write in the terminal: `curl https://device.nais.io/install.sh | bash` \(you might be prompted for your laptop-account's password to unlock `sudo`\).
-3. Turn on your freshly installed `naisdevice` application.
+2. Add the nais PPA repo:
+``` 
+curl -s --compressed "https://ppa.nais.io/KEY.gpg" | sudo apt-key add -
+sudo curl -s --compressed -o /etc/apt/sources.list.d/my_list_file.list "https://ppa.nais.io/my_list_file.list"
```

Comment:
Denne fungerer ikke for meg: https://ppa.nais.io/my_list_file.list <-- 404

Kunne jeg uansett foreslå:
```suggestion
curl -s --compressed -o "https://ppa.nais.io/my_list_file.list" | sudo tee /etc/apt/sources.list.d/my_list_file.list
```